### DISCUSSION TITLE
demo account: verify key?

### DISCUSSION THREAD

**Discussion by nschloe** [keygen-sh/community#57] [Q&A] [answered]
From [here](https://github.com/keygen-sh/example-python-offline-validation-caching/issues/2):

For testing, does the demo account have a public Ed25519 verify key?

**Comment by ezekg** [comment, marked as answer]
I don't recommend using the demo account to run tests against. You should use your own account for that. The demo account and its resources are public and can be viewed by simply visiting https://app.keygen.sh/demo.

Regardless, this is the demo account's Ed25519 public key:

```
e8601e48b69383ba520245fd07971e983d06d22c4257cfd82304601479cee788
```

**Reply by nschloe** [reply]
Thanks for the info!
